etc/pipeline-build/python/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python:3
 
 WORKDIR /app
-ADD build.sh .
-ADD run.sh .
+ADD etc/pipeline-build/python/build.sh .
+ADD etc/pipeline-build/python/run.sh .
```

Comment:
same comment as above: wouldn't a better change be to make whatever does `docker build` of this first `cd` into `etc/pipeline-build/go`? So that anyone can build this docker file from that directory.